SECURITY.md

```diff
@@ -1,22 +0,0 @@
-# Security Policy
```

Comment:
Was this removed by a reason?

Reply:
absolutely no idea - maybe a wrongly staged file in a merge or so ... but its back now https://github.com/pimcore/studio-ui-bundle/pull/549/commits/209fb36b65db1906beb7259cae2b309b74be5bc1